server/data/educationAndWorkPlanClient.ts

```diff
@@ -109,16 +110,27 @@ export default class EducationAndWorkPlanClient {
     prisonNumber: string,
     createInductionRequest: CreateInductionRequest,
     token: string,
-  ): Promise<never> {
+  ): Promise<void> {
     return EducationAndWorkPlanClient.restClient(token).post({
       path: `/inductions/${prisonNumber}`,
       data: createInductionRequest,
     })
   }
 
-  async getEducationResponse(prisonNumber: string, token: string): Promise<EducationResponse> {
+  async getEducation(prisonNumber: string, token: string): Promise<EducationResponse> {
```

Comment:
Minor tidying of this method name, just to make it consistent with the other method names in the client classes